samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -151,7 +179,7 @@ static void fp_account_key_written(struct bt_conn *conn)
 {
 	LOG_INF("Fast Pair: Account Key write");
 
-	app_fp_adv_mode_set(APP_FP_ADV_MODE_NOT_DISCOVERABLE);
+	app_fp_adv_request(&fp_adv_trigger_fmdn_provisioning, true);
```

Comment:
This part can now be placed in the `if` statement as the FMDN provisioning is only triggered for the first Account Key write.

If you can change the code underneath in the following way, it would also be an improvement:

```
	/* The first and only Account Key write starts the FMDN provisioning. */
	if (!fp_account_key_present) {
		/* Fast Pair Implementation Guidelines for the locator tag use case:
		* trigger the reset to factory settings if there is no FMDN
		* provisioning operation within 5 minutes.
		*/
		fmdn_factory_reset_schedule(
			FACTORY_RESET_TRIGGER_PROVISIONING_TIMEOUT,
			K_MINUTES(FMDN_PROVISIONING_TIMEOUT));

```